pype/plugins/global/publish/extract_burnin.py

```diff
@@ -65,6 +65,10 @@ def process(self, instance):
         for i, repre in enumerate(instance.data["representations"]):
             self.log.debug("__ i: `{}`, repre: `{}`".format(i, repre))
 
+            if repre.get("multipartExr", False) is True:
```

Comment:
shouldn't we do this with `representation['tag']` to keep it consistent with other switches?